packages/client/docusaurus/docs/client/05-advanced/03-permissions-and-capabilities.mdx

```diff
@@ -3,4 +3,147 @@ id: 03-permissions-and-capabilities
 title: Permissions and Capabilities
 ---
 
-Stay tuned for more information on this topic.
+import SDKSpecific from '../SDKSpecific';
+
+## Introduction
+
+In some types of calls, there is a common requirement for participant moderation.
```

Comment:
Is participant moderation the correct term here? 

When I glance over it, I'd connect it to content moderation, so kicking users out if necessary. But the permissions and capabilities are much more than that.

Maybe we could go the route of:

```suggestion
In many types of calls, there is a requirement for providing different users with certain permissions and capabilities.
```

Reply:
Good point... we keep referring to this feature as "moderation" but, this can be understood differently but our audience. I'll try to avoid using the term "moderation" from now on.